source/mapping-rules/field-customizations/field-customizations.txt

```diff
@@ -0,0 +1,71 @@
+.. _rm-field-customizations:
+
+====================
+Field Customizations
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Field customizations are optional components of your mapping rules that 
```

Comment:
Add rename fields from the edit rules content to the Field Customizations drawer reference page.